CIP-0010/registry.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
If we want to have a machine-readable format, probably we should get rid of the human-readable table to avoid duplication

Reply:
I agree. Both that a machine readable format is sensible, and also that we should avoid the opportunity for inconsistency.

Reply:
You are right, I'm going to remove it.